### PR TITLE
Add transcription job

### PR DESCRIPTION
**Pull request by kernicPanel**
As transcription generation has been added to Peertube, we want to implement it.

### CODE REVIEW

File: src/django_peertube_runner_connector/utils/job_handlers/transcription_job_handler.py [hunk 1]
```diff
@@ -0,0 +1,128 @@
...
+
+class TranscriptionJobHandler(AbstractJobHandler):
+    """Handler for transcription jobs."""
+
+    def is_abort_supported(self):
+        """Return True if the job supports aborting."""
+        return True
+
+    def specific_abort(self, runner_job: RunnerJob):
+        """Abort the runner job."""
```
[the remark this answers is not in the document]
Comment (kernicPanel) [reply src/django_peertube_runner_connector/utils/job_handlers/transcription_job_handler.py:40]: No pass is needed when a docstring is present.

[the remark this answers is not in the document]
Comment (lunika) [reply src/django_peertube_runner_connector/utils/job_handlers/transcription_job_handler.py:40]: I know but should not be a good practice to do it ?

[the remark this answers is not in the document]
Comment (kernicPanel) [reply src/django_peertube_runner_connector/utils/job_handlers/transcription_job_handler.py:40]: No, pylint has a rule for it. https://pylint.pycqa.org/en/latest/user_guide/messages/warning/unnecessary-pass.html

[the remark this answers is not in the document]
Comment (lunika) [reply src/django_peertube_runner_connector/utils/job_handlers/transcription_job_handler.py:40]: ok, sometimes I don't like pylint.

File: src/django_peertube_runner_connector/utils/job_handlers/transcription_job_handler.py [hunk 2]
```diff
@@ -0,0 +1,128 @@
...
+            domain=domain,
+            job_type=RunnerJobType.VIDEO_TRANSCRIPTION,
+            job_uuid=job_uuid,
+            payload=payload,
+            private_payload=private_payload,
+            priority=0,
+            depends_on_runner_job=None,
+        )
+
+        video.increase_or_create_job_info("pendingTranscript")
```
Comment (lunika) [inline src/django_peertube_runner_connector/utils/job_handlers/transcription_job_handler.py:98]: What do you think about creating a constant for this ? or maybe `VideoJobInfoColumnType.PENDING_TRANSCRIPT` ?

Comment (kernicPanel) [reply src/django_peertube_runner_connector/utils/job_handlers/transcription_job_handler.py:98]: I already created `VideoJobInfoColumnType.PENDING_TRANSCRIPT` but forgot to use it here

**Review by lunika — APPROVED** [review]

**Pull request merged**